lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,52 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
```

Comment:
Yes, that's how we tend to solve that problem (and the warning will actually be removed in some future version of Ruby). It's a well established convention within thoughtbot to do this, at least, and so I think that's what we'd prefer to do here.